src/index.js

```diff
@@ -138,14 +123,14 @@ $.ajax({
     .append('<button id="submit-response" class="ui primary button">Submit response</button>')
 
   // Is case all questions have been responded
-  if (responseCount === questions.length) {
+  if (responseCount == questions.length) {
```

Comment:
You don't need this.
Remember that with "==", javascript actually does more work as it converts the value to same type.

You can check it here: https://jsperf.com/triple-equals-vs-twice-equals